networkcoordinator/networkcoordinator_test.go

```diff
@@ -0,0 +1,87 @@
+/*
+ *    Copyright 2018 Insolar
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package networkcoordinator
+
+import (
+	"context"
+	"testing"
+
+	"github.com/insolar/insolar/core"
+	"github.com/insolar/insolar/core/reply"
+	"github.com/insolar/insolar/log"
+	"github.com/insolar/insolar/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewNetworkCoordinator(t *testing.T) {
+	nc, err := New()
+	require.NoError(t, err)
+	require.Equal(t, &NetworkCoordinator{}, nc)
+}
+
+func TestNetworkCoordinator_Start(t *testing.T) {
+	nc, err := New()
+	require.NoError(t, err)
+	mb := testutils.NewMessageBusMock(t)
+	mb.MustRegisterFunc = func(p core.MessageType, handler core.MessageHandler) {
+		require.Equal(t, p, core.TypeNodeSignRequest)
+	}
+	nc.MessageBus = mb
+	ctx := context.Background()
+	err = nc.Start(ctx)
+	require.NoError(t, err)
```

Comment:
Check that `zeroCoordinator` and `realCoordinator` not empty, and `isStarted` is true